src/utils/constants.ts

```diff
@@ -96,7 +96,7 @@ export default {
   LABEL_CONSOLIDATION_DISAGREE: 'Disagree',
   LABEL_CONSOLIDATION_AGREEMENT: 'Agreed',
   LABEL_CONSOLIDATION_DISAGREEMENT: 'Disagreed',
-  LABEL_DASHBOARD_NO_APPLICATIONS: "You don't have any %1 yet.",
+  LABEL_DASHBOARD_NO_APPLICATIONS: "You don't have any applications in progress",
```

Comment:
Are we sure "in progress" is correct? Doesn't this show up if you have no applications *at all*? (Not sure, just checking)